kiwi/defaults.py

```diff
@@ -564,6 +564,15 @@ def get_shim_loader(root_path):
             for shim_file in glob.iglob(root_path + shim_file_pattern):
                 return shim_file
 
+    @staticmethod
+    def get_distribution_grub_font_directory(root_path):
+        font_dir_patterns = [
+            '/boot/efi/EFI/*/fonts'
```

Comment:
In Fedora, the fonts are stored in `/usr/share/grub`. Why? No idea. @martinezjavier might know why it's `/usr/share/grub` instead of `/usr/share/grub2`...

Reply:
Yes we do search it there. Maybe the name of the method is misleading. This method looks up the distribution specific font dir for grub at boot time. Fedora efi binaries looks up the font in ```/boot/efi/EFI/fedora/fonts```. whereas suse efi binaries looks it up in ```/boot/grub2/fonts```

I treat ```/boot/grub2/fonts``` to be the standard path which should be used by efi binaries as well as by x86 boot code. For the suse systems I know this is true

On Fedora only the x86 boot code searches the fonts in ```/boot/grub2/fonts``` but the efi binaries looks in  ```/boot/efi/EFI/fedora/fonts```

```
rpm -ql grub2-efi-x64

/boot/efi/EFI/fedora/fonts
/boot/efi/EFI/fedora/grub.cfg
/boot/efi/EFI/fedora/grubenv
/boot/efi/EFI/fedora/grubx64.efi
/boot/grub2/grubenv
/boot/loader/entries
/etc/grub2-efi.cfg
```

So kiwi should have a way to provide the font in the "standard" path and additionally provide it to an eventually existing distribution specific path.

@Conan-Kudo if you can come up with a more clear name on the method to somehow make that clear I would love to take it, it was already quite hard to find some :)

Reply:
Especially for images that are multiboot capable (x86 bios code plus efi) the different font lookup is not nice. But we can handle it here

Reply:
I'm testing on a Fedora30 VM in Azure

Reply:
`get_distribution_grub_font_directory` -> `get_grub_efi_font_directory`

Reply:
ok let's give it the efi scope. I hope this will only be needed in that area and is not different for x86 boot code

Reply:
@Conan-Kudo hope it makes more sense now